source/items/tagtool.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Tag Tool
+layout: default
+---
+# Tag Tool
+
+<div class="infobox box text-center">
+    <p style="text-align:center;"><img src="../../assets/images/icons/minecolonies/tag_tool.png" alt="Tag tool"></p>
+</div>
+<br>
+
+The Tag Tool is a creative-only item used to help build custom [schematics](../../source/tutorials/schematics). It allows "tags" (specific words or phrases) to be attached to specific blocks in the schematic. Any tag can be attached to any block, but only specific ones will do something. Some building types require specific tags in order to function correctly, while others are optional.
+
+## Using the Tag Tool
+
+1. Shift-right-click the anchor block of the schematic that you want to view or edit. This will highlight it with a red border and make the associated tags visible.
+
+    * For a building with a hut block, the hut block is the anchor block.
+    * For an upgradeable decoration, the [decoration controller](../../source/items/decocontroller) is the anchor block.
+    * For any other decoration, you must place a Tag Anchor Block as the anchor block.
+
+2. While you are holding the tag tool, you will see all existing tags overlaying their blocks in-world, and they will be highlighted with a white border.
+
+3. Right-click the tag tool in the air to open its GUI. This allows you to type in the tag that you want to apply or remove.  Close the GUI afterwards. The GUI also shows a list of all currently tagged blocks.

```

Comment:
```suggestion
3. Right-click the tag tool in the air to open its GUI. This allows you to type in the tag that you want to apply or remove. Close the GUI afterwards. The GUI also shows a list of all currently tagged blocks.

```
one minor nitpick

Reply:
Double spacing is the One True Way™.  You younguns that didn't grow up with typewriters (grumble grumble)